devops/scripts/benchmarks/main.py

```diff
@@ -423,6 +423,12 @@ def validate_and_parse_env_args(env_args):
         help="Directory for cublas library",
         default=None,
     )
+    parser.add_argument(
+        "--hip_arch",
```

Comment:
All the other options use `-` instead of `_`. can you replace that? Please also fix the cuda-specific arguments.